go.mod

```diff
@@ -4,10 +4,11 @@ go 1.18
 
 require (
 	github.com/CosmWasm/wasmd v1.0.0
-	github.com/cosmos/cosmos-sdk v0.45.12
+	github.com/cosmos/cosmos-sdk v0.45.13-ics
```

Comment:
Can we pump to 0.45.14-ics?

Reply:
interchain-seurity v1.0.0 use v0.45.13-ics: https://github.com/cosmos/interchain-security/blob/v1.0.0/go.mod#L139, so we do not need to pump to 0.45.14-ics to avoid error

Reply:
oh, cool!